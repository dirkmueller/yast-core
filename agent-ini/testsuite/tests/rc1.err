tests/rc1.ycp:6: Wrong value for path .INVALID: 4.5
[YCP] tests/rc1.ycp:XXX english
