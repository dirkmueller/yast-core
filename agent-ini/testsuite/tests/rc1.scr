.

`ag_ini(
  SCR::SysConfigFile ("tests/rc1.in.test")
)
