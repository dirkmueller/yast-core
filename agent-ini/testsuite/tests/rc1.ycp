{
    SCR::Write (.NEW_KEY, "value of the new key");
    SCR::Write (.NEW_KEY.comment, "\n#comment of the new key\n#\n");
    SCR::Write (.ENABLE_SUSECONFIG, nil);
    SCR::Write (.NUMERIC, 5);
    SCR::Write (.INVALID, 4.5);
    y2error ("%1", SCR::Read (.DEFAULT_LANGUAGE));
    return SCR::Dir (.);
}
