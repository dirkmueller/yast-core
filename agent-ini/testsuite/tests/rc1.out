(["DEFAULT_LANGUAGE", "SEPARATED", "SEP2", "ALSO", "NEW_KEY", "NUMERIC"])
#
# This language is used in YaST. This is also the default
# for local users, if RC_LANG and RC_LC_ALL are not set. Root
# uses this language if ROOT_USES_LANG is set to "YES"
#
DEFAULT_LANGUAGE="english"
#
# ...
SEPARATED="abc abc abc cba cba cba"
SEP2="huhu \
      hoho \
      hehe "
ALSO=' apostrophes are allowed '

#comment of the new key
#
NEW_KEY="value of the new key"
NUMERIC="5"
